app/src/js/components/Executions/index.js

```diff
@@ -14,38 +14,30 @@ import { strings } from '../locale';
 
 class Executions extends React.Component {
   query () {
-    this.props.dispatch(getCount({
+    const { dispatch } = this.props;
+    dispatch(getCount({
       type: 'executions',
       field: 'status'
     }));
-    this.props.dispatch(listExecutions());
-    this.displayName = strings.executions;
```

Comment:
Do we need to keep `displayName`?

Reply:
No, usually it would be in the constructor, so I don't think it's doing anything in the `query` function. Also, since the name of the component (Executions) is the same as `string.executions`, I figured it was safe to remove completely.